src/main/java/org/broadinstitute/hellbender/engine/LocusWalker.java

```diff
@@ -74,6 +80,19 @@ public boolean includeNs() {
     }
 
     /**
+     * Does this tool emit information for uncovered loci? Tools that do should override to return {@code true}.
+     *
+     * NOTE:  Typically, this should only be used when intervals are specified.
+     * NOTE:  If MappedReadFilter is removed, then emitting empty loci will fail.
+     *
+     * @return {@code true} if this tool requires uncovered loci information to be emitted, {@code false} otherwise
+     */
+    public boolean emitEmptyLoci() {
+        return false;
+    }
+
+
+    /**
      * Returns default value for the {@link #maxDepthPerSample} parameter, if none is provided on the command line.
      * Default implementation returns 0 (no downsampling by default).
```

Comment:
Actually, going to long is not trivial.

Also, this is actually number of reads per sample-locus, which means we need to go to long if we have any loci w/ coverage > 2 billion...  Not only that, this would be post downsampling.  So, we could (possibly) still support > 2 billion so long as that was not downsampled.

Rolling this back to int.  @vruano, is this okay?  Lots of other stuff will break anyway, if we were to need a long here.  (Per conversation with @lbergelson )